ui/client/src/components/TransportPeer.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright © 2024 Kaleido, Inc.
```

Comment:
should header now be `2025`?

Reply:
Excellent point, I'll update the headers